### PR TITLE
Add new attributes to authorizationpolicy

### PR DESCRIPTION
**Pull request by abhijeetsinha**
https://microsoftgraph.visualstudio.com/onboarding/_git/onboarding/pullrequest/4169?path=%2Freviews%2F4002-add-more-authorizationpolicy-properties%2Fapi.md&_a=files

### CODE REVIEW

File: api-reference/beta/api/authorizationpolicy-update.md [hunk 1]
```diff
@@ -140,3 +144,59 @@ The following is an example of the response.
 ```http
 HTTP/1.1 204 No Content
 ```
+### Example 4: Disable default user role's permission to create applications
+
+#### Request
+
+The following is an example of the request.
+
```
Comment (Lauragra) [inline api-reference/beta/api/authorizationpolicy-update.md:152]: Please add the HTML comments to the request.

File: api-reference/beta/api/authorizationpolicy-update.md [hunk 2]
```diff
@@ -140,3 +144,59 @@ The following is an example of the response.
...
+```http
+HTTP/1.1 204 No Content
+```
+
+### Example 5: Enable default user role to use Self-Serve Password Reset feature
+
+#### Request
+
+The following is an example of the request.
+
```
Comment (Lauragra) [inline api-reference/beta/api/authorizationpolicy-update.md:182]: Here too.

File: api-reference/beta/api/authorizationpolicy-update.md (hunk 1 above)
Comment (abhijeetsinha) [reply api-reference/beta/api/authorizationpolicy-update.md:167]: fixed

File: api-reference/beta/api/authorizationpolicy-update.md (hunk 2 above)
Comment (abhijeetsinha) [reply api-reference/beta/api/authorizationpolicy-update.md:202]: fixed

**Comment by abhijeetsinha** [comment]
@Lauragra - I have taken care of your feedback. Can you please merge it now?

**Pull request merged**